src/database/schema.zmodel

```diff
@@ -349,9 +349,47 @@ model EventComment {
   @@map("event_comments")
 }
 
+enum recurringType{
+  DAILY
+  WEEKLY
+  MONTHLY
+  YEARLY
+}
+
+model RecurringEvent{
+  id String @id() @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  separationCount Int @map("separation_count")
+  recurringType recurringType @map("recurring_type")
+  title String @db.VarChar(255)
+  titleEn String? @map("title_en") @db.VarChar(255)
+  description String
+  descriptionEn String? @map("description_en")
+  link String? @db.VarChar(255)
+  location String? @db.VarChar(255)
+  organizer String @db.VarChar(255)
+  authorId String @map("author_id") @db.Uuid
+  shortDescription String? @map("short_description") @db.VarChar(255)
+  shortDescriptionEn String? @map("short_description_en") @db.VarChar(255)
+  startDatetime DateTime @map("start_datetime") @db.Timestamptz(6)
+  endDatetime DateTime @map("end_datetime") @db.Timestamptz(6)
+  numberOfUpdates Int? @default(0) @map("number_of_updates")
+  slug String? @unique(map: "recurring_events_slug_unique") @db.VarChar(255)
+  alarmActive Boolean? @default(false) @map("alarm_active")
+  tags Tag[] @relation("recurring_event_tags")
+  events Event[]
+  author Member @relation(fields: [authorId], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "events_author_id_foreign")
+
+  @@allow("create", has(auth().policies, "event:create"))
+  @@allow("read", has(auth().policies, "event:read"))
+  @@allow("update", has(auth().policies, "event:update"))
+  @@allow("delete", has(auth().policies, "event:delete"))
+}
```

Comment:
Just looking through this PR real quick and I don't think we should have this kind of duplication, at least I think a lot of these fields are just copy-pasted from the Event model. Instead I think this should inherit from events, adding some extra fields necessary for repeated events.

Reply:
Yeah you're probably right. My idea with this originally was that recurring events would read from the event in the RecurringEvent table, so that updating for example the title for all events would just be changing this one event. I didn't end up doing it this way though so yes I should remove a lot of the fields.